### ISSUE TITLE
Playlist - new file is not played when "resume playback at last position" selected

### ISSUE THREAD

**Issue by sciguyryan** [zachsaw/MPDN_Extensions#232]
Good afternoon.

This is a follow up filed per the discussion on the Doom9 thread.

When the option to resume playback at last position is selected, attempting to open a new file will re-open the file from the playlist and not the new file.

I'm not sure if this is intended or not but it seems quite counter intuitive.

**Comment by Garteal** [comment]
Are you opening the file from MPDN or the playlist?

Note that we both have "add" file and "open" file in the playlist. "Add" simply appends to the playlist and "open" clears the playlist, adds the selected file(s) and begins playback. "Open" is currently only available via the context menu. (Right-click anywhere in the Playlist)

**Comment by sciguyryan** [comment]
From shell actually. Not from the player at all. This would be from an associated file or whatnot.

**Comment by Garteal** [comment]
Okay. Do you have "Begin playback on startup" checked? If so then the Playlist will want to continue playing whatever was playing before MPDN was closed.

I'll take a look to see if this can be fixed, but in the meantime you can turn off the option. Note that since this is seen as opening a file, it'll clear the Playlist and add and play what you open.

**Comment by sciguyryan** [comment]
That's correct. I do. Though I filed this mainly because I noted it was handled more elegantly in other players.

If it can't be fixed then there are plenty of ways to work around it, I just wanted to file it in case.

Cheers.
